### PR TITLE
Don't assume locale is provided when converting from Omnipay format to Klarna format

### PR DESCRIPTION
**Pull request by fred-jan**
The `getOrderData` protected function is also used by `UpdateTransactionRequest`, where the locale is not required. Currenly `UpdateTransactionRequest` triggers an error, because it always assumes there's a locale provided.

### CODE REVIEW

File: src/Message/AbstractOrderRequest.php [hunk 1]
```diff
@@ -156,14 +156,17 @@ public function setWidgetOptions($widgetOptions)
...
         $data = [
-            'locale' => str_replace('_', '-', $this->getLocale()),
             'order_amount' => $this->getAmountInteger(),
             'order_tax_amount' => $this->toCurrencyMinorUnits($this->getTaxAmount()),
             'order_lines' => $this->getItemData($this->getItems()),
-            'purchase_country' => explode('_', $this->getLocale())[1],
             'purchase_currency' => $this->getCurrency(),
         ];
 
+        if (null !== $locale = $this->getLocale()) {
```
Comment (digibeuk) [inline src/Message/AbstractOrderRequest.php:165]: you should add a test path for the situation when this is not null

**Review by digibeuk — CHANGES_REQUESTED** [review]

File: src/Message/AbstractOrderRequest.php (hunk 1 above)
Comment (fred-jan) [reply src/Message/AbstractOrderRequest.php:165]: The test path for this situation were already there (it was mandatory until this moment). I've altered an existing test instead to check the situation when a locale has been omitted.

**Review by digibeuk — APPROVED** [review]
